pages/categories/index.js

```diff
@@ -0,0 +1,51 @@
+import StyledBackLink from "@/components/StyledBackLink";
+import BackArrow from "@/public/assets/images/back-arrow.svg";
+import styled from "styled-components";
+import { StyledMessage } from "..";
+import Modal from "@/components/Modal";
+import CategoriesList from "@/components/CategoriesList";
+import CategoryForm from "@/components/CategoryForm";
+import Plus from "@/public/assets/images/plus.svg";
+
+const StyledHeading = styled.h2`
+  text-align: center;
+`;
+
+const StyledPlus = styled(Plus)`
+  position: fixed;
+  bottom: 4rem;
+  right: calc(50% - 160px);
+  width: 3rem;
+  fill: grey;
+`;
+
+export default function CategoriesPage({
+  categories,
+  onAddCategory,
+  showModal,
+  setShowModal,
+  familyMembers,
+}) {
+  return (
+    <>
+      <StyledHeading>Task Categories</StyledHeading>
+
+      {!categories.length && (
+        <StyledMessage>The list is empty. Add members to begin!</StyledMessage>
+      )}
+      <CategoriesList categories={categories} />
+
+      <StyledPlus onClick={() => setShowModal(true)} $right={true} />
```

Comment:
you could use the shorthand for booleans props here:

```suggestion
      <StyledPlus onClick={() => setShowModal(true)} $right />
```

Reply:
thank you for the review...will consider for further implementation